src/fides/api/models/privacy_experience.py

```diff
@@ -141,67 +392,158 @@ def dry_update(self, *, data: dict[str, Any]) -> PrivacyExperienceConfig:
+                for key, val in translation_data.items():
+                    cloned_attributes[key] = val
+                cloned_attributes.pop("_sa_instance_state")
+                cloned_attributes.pop("id")
+            else:
+                cloned_attributes = translation_data
+            dry_updated_translations.append(ExperienceTranslation(**cloned_attributes))
+        return dry_updated_translations
+
+
+class ExperienceTranslation(ExperienceTranslationBase, Base):
+    """Stores all the translations for a given Experience Config"""
+
+    experience_config_id = Column(
+        String, ForeignKey(PrivacyExperienceConfig.id_field_path), nullable=False
+    )
+
+    __table_args__ = (
+        UniqueConstraint(
+            "language", "experience_config_id", name="experience_translation"
+        ),
+    )
+
+    histories: RelationshipProperty[AppenderQuery] = relationship(
+        "PrivacyExperienceConfigHistory",
+        backref="experience_translation",
+        lazy="dynamic",
+        order_by="PrivacyExperienceConfigHistory.created_at",
+    )
+
+    @property
+    def version(self) -> Optional[float]:
+        """Convenience property that returns the latest version number of the translation"""
+        return (
+            self.experience_config_history.version  # type: ignore[return-value]
+            if self.experience_config_history
+            else None
+        )
+
+    @property
+    def experience_config_history(self) -> Optional[PrivacyExperienceConfigHistory]:
+        """Convenience property that returns the experience config history for the latest version.
+
+        Note that there are possibly many historical records for the given experience config translation,
+        this just returns the current corresponding historical record.
+        """
+        # Histories are sorted at the relationship level
+        return self.histories[-1] if self.histories.count() else None
+
+    @property
     def experience_config_history_id(self) -> Optional[str]:
-        """Convenience property that returns the experience config id for the current version.
+        """Convenience property that returns the experience config history id for the latest version.
 
-        Note that there are possibly many historical records for the given experience config, this just returns the current
+        Note that there are possibly many historical records for the given experience config translation, this just returns the current
         corresponding historical record.
         """
-        history: PrivacyExperienceConfigHistory = self.histories.filter_by(  # type: ignore # pylint: disable=no-member
-            version=self.version
-        ).first()
-        return history.id if history else None
-
-    @classmethod
-    def get_default_config(
-        cls, db: Session, component: ComponentType
-    ) -> Optional[PrivacyExperienceConfig]:
-        """Load the first default config of a given component type"""
```

Comment:
This concept has been removed, the "default" Experience Config.  We used to ship with one default Config of each Component Type, but now we ship with many ExperienceConfigs, there are no "default" types.